packages_rs/nextclade/src/align/seed_alignment.rs

```diff
@@ -208,6 +210,10 @@ pub fn create_stripes(
   for slice in shifts.windows(4) {
     let min = slice.iter().min().unwrap();
     let max = slice.iter().max().unwrap();
+    let width = max - min;
+    if width as usize> max_indel {
```

Comment:
Max indel is now not just a single indel but the total shift over 3*seed spacing. So we may want to increase max_indel by default.